flagging_site/data/usgs.py

```diff
@@ -4,8 +4,67 @@
 
 https://waterdata.usgs.gov/nwis/uv?site_no=01104500
 """
+import pandas as pd
+import requests
+import io
 
 # Constants
-# ~~~~~~~~~
+USGS_URL = 'https://waterdata.usgs.gov/nwis/uv?site_no=01104500'
```

Comment:
This is good to have for documentation purposes (I see it in the module docstring-- very good, I like that!), but the `USGS_URL` should be `https://waterservices.usgs.gov/nwis/iv/`. This variable should be the URL used in the API call in the function.